pkgs/development/libraries/howard-hinnant-date/default.nix

```
@@ -19,6 +19,13 @@ stdenv.mkDerivation rec {
       url = "https://git.alpinelinux.org/aports/plain/community/date/538-output-date-pc-for-pkg-config.patch?id=11f6b4d4206b0648182e7b41cd57dcc9ccea0728";
       sha256 = "1ma0586jsd89jgwbmd2qlvlc8pshs1pc4zk5drgxi3qvp8ai1154";
     })
+    # Fix CMake include directory path.
+    # https://github.com/HowardHinnant/date/pull/753
+    (fetchpatch {
+      name = "fix-cmake-include-dir.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/HowardHinnant/date/pull/753.patch";
```

```suggestion
      url = "https://github.com/HowardHinnant/date/commit/8061b53c489b0c8676feedcb65049b27664327b5.patch";
```
better to use a permalink in case the PR changes